README.md

```diff
@@ -48,3 +44,15 @@ Example file to place at `~/.puppetlabs/client-tools/puppetdb.conf`:
     }
 
 ~~~
+
+## Install from source
+
+Make sure an appropriate version of leatherman is installed on your system.
+
+~~~bash
+
+    $ mkdir build && cd build
+    $ CMAKE_PREFIX_PATH=/usr/local/opt/curl/lib cmake ..
```

Comment:
the curl/lib stuff is just for OSX and if that's where you have your Homebrew setup, right?